Merge 19.0 code freeze into `develop`

mokagio opened the pull request.

mokagio committed:
Bump version number

mokagio committed:
Update draft release notes for 19.0.

mokagio committed:
Release Notes: add new section for next version (19.1)

mokagio committed:
Use stable version of internal pods for version 19.0

mokagio committed:
Update strings for localization

mokagio reviewed (commented):
- [x] Switch to production Pods
- [x] New version header in `RELEASE-NOTE.txt` 
- [x] `Localizable.strings` updated 
- [x] `release_notes.txt` updated with notes from `RELEASE-NOTE.txt` for current version 
- [x] Version update in `.xcconfig` and `download_metadata.swift`

AliSoftware commented on WordPress/Resources/release_notes.txt:
```diff
@@ -1,7 +1,9 @@
-New year, new look! We updated comment threads with a shiny new design for author labels, accessory and share buttons, and row and reply highlighting. We also made a few updates to the Block Editor.
+* [**] Video uploads: video upload is now limited to 5 minutes per video on free plans.
+* [*] Block editor: Give multi-line block names central alignment in inserter [https://github.com/wordpress-mobile/gutenberg-mobile/pull/4343]
+* [**] Block editor: Fix missing translations by refactoring the editor initialization code [https://github.com/wordpress-mobile/gutenberg-mobile/pull/4332]
+* [**] Block editor: Add Jetpack and Layout Grid translations [https://github.com/wordpress-mobile/gutenberg-mobile/pull/4359]
+* [**] Block editor: Fix text formatting mode lost after backspace is used [https://github.com/wordpress-mobile/gutenberg-mobile/pull/4423]
+* [*] Block editor: Add missing translations of unsupported block editor modal [https://github.com/wordpress-mobile/gutenberg-mobile/pull/4410]
+
+* [**] Time zone suggester: we have a new time zone selection screen that suggests the time zone based on the device, and improves search.
```
Speaking of pdnsEh-6h-p2 you should copy this file to `WordPress/Jetpack/Resources/release_notes.txt`.

This would both ensure that we don't forget to update the file, but also be less confusing when we point our freelancer to those files to translate release notes for WP and JP (even if this time around they might be the exact same, that won't always be the case)

mokagio committed:
Update Jetpack `release_notes.txt` source for version 19.0

Thanks @AliSoftware.

See
https://github.com/wordpress-mobile/WordPress-iOS/pull/17732#discussion_r781110071
